platforms/native/engine/jsb-fs-utils.js

```diff
@@ -23,9 +23,11 @@ let jsb_downloader = null;
 const downloading = new cc.AssetManager.Cache();
 let tempDir = '';
 
-jsb.Downloader.prototype._ctor = function () {
-    this.__nativeRefs = {};
-};
+if(!window.oh) {
+    jsb.Downloader.prototype._ctor = function () {
```

Comment:
Should add TODO to support, or will not support on OH?

Reply:
This part has not yet been adapted

Reply:
Ok, then please add TODO comment.

Reply:
Ok